QUICKSTART.md

```diff
@@ -0,0 +1,308 @@
+`opentelemetry-sdk` package, or any other implementation of the OpenTelemetry API. This way, libraries will obtain a real tracer
+implementation only if the user application is configured for it. For more details, check out the [Library Guidelines].
+
+[Library Guidelines]: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-guidelines.md
+
+The application has to install a span processor with an exporter and may customize the behavior of the OpenTelemetry SDK.
+
+For example, a basic configuration instantiates the SDK tracer registry and sets to export the traces to a logging stream.
+
+```java
+// Get the tracer
+TracerSdkRegistry tracerRegistry = OpenTelemetrySdk.getTracerRegistry();
+
+// Set to export the traces to a logging stream
+tracerRegistry.addSpanProcessor(SimpleSpansProcessor.newBuilder(new LoggingExporter()).build());
+```
+
+### Sampler
+
+It is not always feasible to trace and export every user request in an application.
+In order to strike a balance between observability and expenses, traces can be sampled. 
+
+The OpenTelemetry SDK offers three samplers out of the box:
+ - Always sample
+ - Never sample
+ - Sampling based on probability
+
+Additional samplers can be provided implementing the [`io.opentelemetry.sdk.trace.Sampler`] interface.
+
+```java
+TraceConfig alwaysOn = TraceConfig.getDefault().toBuilder().setSampler(
+        Samplers.alwaysOn()
+).build();
+TraceConfig alwaysOff = TraceConfig.getDefault().toBuilder().setSampler(
+        Samplers.alwaysOff()
+).build();
+TraceConfig half = TraceConfig.getDefault().toBuilder().setSampler(
+        Samplers.probability(0.5)
+).build();
+// Configure the sampler to use
+tracerRegistry.updateActiveTraceConfig(
+    half
+);
+```
+
+[`io.opentelemetry.sdk.trace.Sampler`]: https://github.com/open-telemetry/opentelemetry-java/blob/master/sdk/src/main/java/io/opentelemetry/sdk/trace/Sampler.java
+
+### Span Processor
+
+Different Span processors are offered by OpenTelemetry. 
+The `SimpleSpanProcessor` immediately forwards ended spans to the exporter, while the `BatchSpansProcessor` batches them and sends them in bulk.
+Multiple Span processors can be configured to be active at the same time using the `MultiSpanProcessor`.
+
+```java
+tracerRegistry.addSpanProcessor(
+    SimpleSpansProcessor.newBuilder(new LoggingExporter()).build()
+);
+tracerRegistry.addSpanProcessor(
+    BatchSpansProcessor.newBuilder(new LoggingExporter()).build()
+);
+tracerRegistry.addSpanProcessor(MultiSpanProcessor.create(Arrays.asList(
+            SimpleSpansProcessor.newBuilder(new LoggingExporter()).build(),
+            BatchSpansProcessor.newBuilder(new LoggingExporter()).build()
+)));
+```
+
+### Exporter
+
+Span Processor are initialized with an exporter which is responsible to send the telemetry data to your backend of choice.
+OpenTelemetry offers four exporters out of the box:
```

Comment:
Let's also add a link over to the OpenTelemetry registry to find other java exporters.

https://opentelemetry.io/registry/?s=exporter

Reply:
Nice idea!